src/client/gui/lib/extensions.dart

```diff
@@ -107,3 +107,45 @@ extension NullableMap<T> on T? {
     }
   }
 }
+
+/// A custom wrapper for SelectableText with a white popup (right-click) menu.
+class WhitePopupMenuSelectableText extends StatelessWidget {
+  final String text;
+  final TextStyle? style;
+  final TextAlign? textAlign;
+  final int? maxLines;
+
+  const WhitePopupMenuSelectableText({
+    super.key,
+    required this.text,
+    this.style,
```

Comment:
Just to keep things in line with the usual text widgets, how about you make `text` a positional parameter instead of a required named parameter?